tests/polyskel_convex_test.py

```diff
@@ -0,0 +1,226 @@
+# coding=utf-8
+"""Classes for computing straight skeleton for 2D polygons."""
+from __future__ import division
+
+import pytest
+
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry2d.pointvector import Point2D
+from ladybug_geometry.geometry2d.line import LineSegment2D
+from ladybug_geometry import polyskel as lb_polyskel
+
+from pprint import pprint as pp
+import numpy as np
+
+# TODO Figure out if we need this later
+# # For comparison
+# import sys
+# paths = ['/app/polyskel/']
+# for path in paths:
+#     if path not in sys.path: sys.path.insert(0,path)
+# import polyskel as orig_polyskel
+
+#TODO: move to updated polyskel once we import lb-geom?
+def skeleton_edges(skeleton):
+    """
+    Consumes list of polyskeleton subtrees.
+    Skeleton edges are the segments defined by source point and each sink points.
+
+    Args:
+        skeleton: list of polyskel.Subtree, which are namedTuples of consisting of
+        a source point, and list of sink points.
+
+    Returns:
+        list of LineSegment2Ds
+    """
+    L = []
+    for subtree in skeleton:
+        source = subtree.source
+        for sink in subtree.sinks:
+            edge = LineSegment2D.from_end_points(
+                Point2D(source[0], source[1]),
+                Point2D(sink[0],sink[1])
+            )
+            L.append(edge)
+    return L
+
+# TODO Make this a from_array() method
+def helper_mtx2line2d(lst):
+    """ List of tuples of point coordinates to LineSegment2D"""
+    p1, p2 = lst[0], lst[1]
+    line2d = LineSegment2D.from_end_points(
+        Point2D(*p1), Point2D(*p2)
+        )
+    return line2d
+
+# TODO Make this a from_array() method
+def helper_mtx2polygon2d(lst):
+    polygon = Polygon2D([Point2D(*pt) for pt in lst])
+    return polygon
+
+
+def helper_assert_polygon_equality(polygon, chk_edges, holes=None, lb=True):

```

Comment:
Docstring here might be helpful if I want to add some tests and I don't know what the input arguments should be.